spec/index.html

```diff
@@ -443,32 +520,68 @@ <h3>Triples</h3>
+      a <a>literal</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
     </ul>
 
     <p>An RDF triple is conventionally written in the order subject,
       predicate, object.</p>
 
+    <p>A <dfn data-local-lt="quoted">quoted triple</dfn> is an <a>RDF triple</a>
+      used as the <a>subject</a> or <a>object</a> of another triple.</p>
+
+    <p>An <dfn data-local-lt="asserted">asserted triple</dfn>
+      is an <a>RDF triple</a> that is an element of an <a>RDF graph</a>.</p>
+
+    <p>A <dfn>complex triple</dfn> is an <a>RDF triple</a>
+      having a <a>quoted triple</a> as a component.</p>
+
+    <p>A <dfn>simple triple</dfn> is an <a>RDF triple</a>
+      that does not have a <a>quoted triple</a> as a component.</p>
+
+    <p>A <dfn class="export">complex graph</dfn> is
+      an <a>RDF graph</a> containing one or more <a>complex triples</a>.</p>
+
+    <p>A <dfn class="export">simple graph</dfn> is
+      an <a>RDF graph</a> composed entirely of <a>simple triples</a>.</p>
+
+    <p>The <dfn data-lt="constituent">constituent terms</dfn> (or simply <a>constituents</a>)
+      of a <a>triple</a>
+      is the set containing its <a>subject</a>, its <a>predicate</a>, its <a>object</a>,
+      plus all the <a>constituent terms</a> of its <a>subject</a> and/or its <a>object</a>
+      if they are themselves <a>quoted triples</a>.
+      By extension, we define the <a>constituent terms</a> of a <a>complex graph</a>
+      to be the union set of the <a>constituent terms</a> of all its triples.</p>
+
     <p>The set of <span id="dfn-nodes"><!-- obsolete term--></span><dfn data-lt="node">nodes</dfn> of an <a>RDF graph</a>
-      is the set of subjects and objects of triples in the graph.
+      is the set of <a>subjects</a> and <a>objects</a> of <a>triples</a> in the graph
+      plus all the <a>nodes</a> of those <a>subjects</a> and/or its <a>objects</a>,
+      recursively, for <a>complex triples</a>.
       It is possible for a predicate IRI to also occur as a node in
       the same graph.</p>
 
-    <p><a>IRIs</a>, <a>literals</a> and
-      <a>blank nodes</a> are collectively known as
+    <p><a>IRIs</a>, <a>literals</a>,
+      <a>blank nodes</a>, and <a>quoted triples</a> are collectively known as
       <span id="dfn-rdf-terms"><!-- obsolete term--></span><dfn data-lt="rdf term">RDF terms</dfn>.</p>
 
-    <p><a>IRIs</a>, <a>literals</a>
-      and <a>blank nodes</a> are distinct and distinguishable.
+    <p><a>IRIs</a>, <a>literals</a>,
+      <a>blank nodes</a>, and <a>quoted triples</a> are distinct and distinguishable.
       For example, <code>http://example.org/</code> as a string literal
       is neither equal to <code>http://example.org/</code> as an IRI,
       nor to a blank node with the <a>blank node identifier</a>
       <code>http://example.org/</code>.</p>
+
+    <p class="note">In a given <a>RDF graph</a>,
+      the same <a>triple</a> MAY be both <a>quoted</a> and <a>asserted</a>.</p>
```

Comment:
Given that we're in an informative note, there is no normative meaning to "MAY", so it can be written "may".

```suggestion
      the same <a>triple</a> may be both <a>quoted</a> and <a>asserted</a>.</p>
```